### PR TITLE
Project: Restaurant Page

### PR DESCRIPTION
**Pull request by OlukaDenis**
- Using Webpack
- Creating DOM elements with Javascript

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        <li id="contact" class="nav-item">
+          <i class="fas fa-headset"></i>
+          <span>
+            <a href="#">Contact</a>
+          </span>
+        </li>
+      </ul>
+    </nav>
+    <div class="content" id="content"></div>
+
```
Comment (bellom) [inline dist/index.html]: I guess you missed this; we are supposed to have just this 👉  `<div class="content" id="content"></div>'

According to the Odin project instructions: 
image
https://www.theodinproject.com/courses/javascript/lessons/restaurant-page

File: package.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    "css-loader": "^3.5.2",
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.1.0",
+    "eslint-import-resolver-webpack": "^0.12.1",
+    "eslint-plugin-import": "^2.20.2",
+    "file-loader": "^6.0.0",
+    "style-loader": "^1.1.4",
+    "url-loader": "^4.1.0",
+    "webpack": "^4.42.1",
+    "webpack-cli": "^3.3.11"
```
Comment (bellom) [inline package.json]: image
You are missing -> `webpack-dev-server`
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/

File: package.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
+{
+  "name": "restaurant-page",
+  "version": "1.0.0",
+  "description": "A website restaturant",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack --mode production",
+    "start": "webpack --watch --mode development"
```
Comment (bellom) [inline package.json]: I couldn't open the server with `npm start`.
Let's try this
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open"
  },
```

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @OlukaDenis  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 
 
**Kindly check out my comments below;**

**Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....**

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Review by jofer86 — APPROVED** [review]
## :wave: :wave: Hello!, I'm Jorge, and  I'm here to review this Pull Request, let's take a look shall we :eye: :eye: :eye: 
- Changes marked as **[OPTIONAL]** are not hard requirements but are nice to have.
- All required changes were tackled!! Great Job!!!
# `status: Approved!!` :+1: :rocket: :rocket: 


--------------------------------------------------------------------
Cheers, 
Jorge Fernando Rincón Figueroa.
Technical Support Engineer.
**Slack: @jofer86**

**Comment by OlukaDenis** [comment]
Thanks

**Pull request merged**